File: Coding Exercises/Python/Day21/find_first_and_last.py
def find_first_and_last(arr, target):
    result = [-1, -1]
    for i in range(len(arr)):
        if arr[i] == target:
            result[0] = i
            break
    for i in range(len(arr)-1, -1, -1):
        if arr[i] == target:
            result[1] = i
            break
    return result

File: Coding Exercises/TypeScript/Day21/findFirstAndLast.ts
function findFirstAndLast(arr: number[], target: number): number[] {
  let result: number[] = [-1, -1];
  for (let i = 0; i < arr.length; i++) {
      if (arr[i] === target) {
          result[0] = i;
          break;
      }
  }
  for (let i = arr.length - 1; i >= 0; i--) {
      if (arr[i] === target) {
          result[1] = i;
          break;
      }
  }
  return result;
}
